sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1504,6 +1504,22 @@ object GpuOverrides {
           GpuPythonUDF(a.name, a.func, a.dataType,
             childExprs.map(_.convertToGpu()),
             a.evalType, a.udfDeterministic, a.resultId)
+
+        override def tagExprForGpu(): Unit = {
+          // Allow array type data only in output for the top Python UDF. Because seems
+          // there is something wrong in writing array type data by cuDF arrow IPC writer.
```

Comment:
nit: Could you file an issue for this?  Ideally we would file the issue and include a link to the issue here so it is clear when we could fix it.  But it is okay to check this in first if it will take some time to create a reproducible use case.

Reply:
Here it is https://github.com/NVIDIA/spark-rapids/issues/1171